### ISSUE TITLE
How to upload Images/Files without Uploadcare?

### ISSUE THREAD

**Issue by b3nd1k** [monicahq/monica#6688]
On Chandler (selfhosted), how do I upload images and files without using Uplodcare? I'd like to keep my files locally.

**Comment by b3nd1k** [comment]
Let's hope it happens soon. Not having the option of uploading files if you want to avoid using Uploadcare is not ideal.

**Comment by phorcys420** [comment]
it's temporary.
https://github.com/monicahq/chandler/issues/346

**Comment by b3nd1k** [comment]
@tamaskan Any updates to this? From what I can read, Chandler is out of the beta is now default. I still can't seem to use local storage.

**Comment by Androz2091** [comment]
I agree, I don't like sending my files to an external service, I would like to make my own backups on my own infrastructure... Especially when they are photos... ! Thank you

**Comment by phorcys420** [comment]
It's been a while since this issue has been opened and I honestly think that it's a shame it's not implemented yet.
This is what's stopping me from daily driving the beta and submitting any bugs/etc, I think the project would gain a lot from this feature (this and reimplementing DAV sync) because a lot more people would want to test it.

I would honestly be fine with replacing Uploadcare with s3-compatible file storage before having a built-in solution, it's easy enough and can run anywhere, thanks.
